models/classes/class.QtiTestService.php

```diff
@@ -1446,21 +1453,33 @@ private function verifyItemPermissions(core_kernel_classes_Resource $oldTest, st
         $this->getSecureResourceService()->validatePermissions($ids, ['READ']);
     }
 
-    /**
-     * @return QtiPackageImportPreprocessing
-     */
-    private function getQtiPackageImportPreprocessing()
+    private function getSubclassesByLabel(core_kernel_classes_Class $root, string $label)
```

Comment:
You can **apply the deletion directly here** instead of having to iterate 2x by the resources here 
https://github.com/oat-sa/extension-tao-testqti/pull/2393/files#diff-05b1f87fb7a793e119a1ba8079639c0bfa12edb96f8e45615c2807361951b09bR881

If you do that, method name needs to be changed